From e0e605348d7ce8ccc9402ae0546e195b161aff58 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Mon, 26 Oct 2026 21:26:28 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh Public-Key from keyservers/WKD: same certificate(s), with self-
signatures our copy was missing.
Release bumped with a changelog entry.

Assisted-by: Claude Opus 5.5
---
 mingw-pcre.spec | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/mingw-pcre.spec b/mingw-pcre.spec
index 131807f..b8ad993 100644
--- a/mingw-pcre.spec
+++ b/mingw-pcre.spec
@@ -7,7 +7,7 @@
 Name:		mingw-%{name1}
 Version:	8.45
 %global myversion %{version}%{?rcversion:-%rcversion}
-Release:	11%{?dist}
+Release:	12%{?dist}
 Summary:	MinGW Windows pcre library
 
 License:	BSD-3-Clause
@@ -38,7 +38,7 @@ BuildArch:	noarch
 BuildRequires:  make
 BuildRequires:  git
 BuildRequires:	redhat-rpm-config
-BuildRequires:  gnupg2
+BuildRequires:  openpgpverify
 BuildRequires:  autoconf
 BuildRequires:  automake
 BuildRequires:  libtool
@@ -113,7 +113,7 @@ Static version of the mingw64-pcre library.
 
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %autosetup -S git_am -n pcre-%{version}
 autoreconf -vif
 
@@ -207,6 +207,9 @@ find $RPM_BUILD_ROOT -name "*.la" -delete
 
 
 %changelog
+* Fri Oct 23 2026 Jakub Jelen <jjelen@redhat.com> - 8.45-12
+- Verify upstream source signatures with openpgpverify
+
 * Thu Jul 16 2026 Fedora Release Engineering <releng@fedoraproject.org> - 8.45-11
 - Rebuilt for https://fedoraproject.org/wiki/Fedora_45_Mass_Rebuild
 
-- 
2.56.0

